ui/hooks/src/utils/entry-utils.ts

```diff
@@ -126,5 +129,6 @@ export const mapBeamEntryData = (
     tags: beam?.tags
       ?.filter(labeledTag => labeledTag.labelType === sdk.services.gql.labelTypes.TAG)
       .map(labeledTag => labeledTag.value),
+    reflectionsCount: beam?.reflectionsCount,
```

Comment:
this number includes all reflections no matter of their state(ex: removed) so it's not accurate

Reply:
we need to track these cases but it should be enough atm